packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -300,6 +298,15 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 		select( PLANS_STORE ).getPlanByProductId( selectedPaidPlanProductId )
 	);
 
+	const { selectedPlanProduct } = useSelect( ( select ) => {
+		const plansStore = select( PLANS_STORE );
```

Comment:
There are 5 calls to `select(PLAN_STORE)` in PlanStep component. Let's follow the `useSelect` base practices in p7H4VZ-2Q6-p2 and 
> Do all selections from the same store in one callback

In addition, we can consider returning `billingPeriod` (or maybe better called `selectedPlanBillingPeriod` ?) instead of the entire `selectedPlanProduct` since we don't need the whole object here.

Reply:
I've grouped 3 of them in 73a7359ac81177d0b6662965947c16ea661709bc 
`allAvailablePlansProducts` has some dependencies so it's still separate and [now it has a dependency array](https://github.com/Automattic/wp-calypso/pull/49412/commits/369b4f8dd90c7e0309d60ab159afbe243afcca8e) as recommended in p7H4VZ-2Q6-p2#comment-9214

Reply:
Thanks for the update! 

> I've grouped 3 of them in 73a7359

Awesome! Makes total sense to group them in a single call.

> `allAvailablePlansProducts` has some dependencies so it's still separate.

Good to know about the second parameter for `useSelect`. Thanks for adding this 👍